### PR TITLE
Mt3 470/create post visit actions

### PR DESCRIPTION
**Pull request by Gweaton**
This is very nearly ready, but for some reason submitting a process is overwriting existing notes, rather than appending them to the `Notes` array

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -356,6 +356,71 @@ router.patch("/v1/processes/:ref/appraise", (req, res) => {
...
+      Authorization: matApiToken,
+      "Content-Type": "application/json",
+    },
+    timeout: 10 * 1000,
+  };
+
+  req.body = {
+    subject: subjectId,
+    source: "1", //leave as 1, maps to "Estate officer" value for hackney_handleby filed
+    processType: "2", //Post visit action
```
Comment (erbridge) [inline server/api.js:403]: Let's pull these out as const variables and remove the comments.

File: server/api.js [hunk 2]
```diff
@@ -356,6 +356,71 @@ router.patch("/v1/processes/:ref/appraise", (req, res) => {
...
+    ServiceRequest: {
+      title: "Tenancy Management", //Any title
```
Comment (erbridge) [inline server/api.js:405]: Is this is any title, let's use the one from the env: `process.env.PROCESS_TYPE_NAME`.

File: components/PostVisitActionInput.tsx [hunk 3]
```diff
@@ -24,7 +24,11 @@ export const PostVisitActionInput: React.FunctionComponent<PostVisitActionInputP
 ) => {
   const { label, name, rows, value, onValueChange, required, disabled } = props;
 
-  const currentNote: Note | undefined = value[0];
+  const noteIndex: number | undefined = value.findIndex(
+    (note) => !note.createdAt
+  );
```
Comment (erbridge) [inline components/PostVisitActionInput.tsx:29]:
```suggestion
  const noteIndex = value.findIndex(
    (note) => !note.createdAt
  ) || 0;
```

**Review by erbridge — APPROVED** [review]
LGTM, assuming you're aware that we're still not rendering pervious PVAs.

**Comment by Gweaton** [comment]
@erbridge Thanks. Displaying PVAs is coming very soon in [MT3-669](https://hackney.atlassian.net/secure/RapidBoard.jspa?rapidView=27&projectKey=MT3&modal=detail&selectedIssue=MT3-669)

**Pull request merged**